tools/generator/cmd/root.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/Azure/azure-sdk-for-go/tools/generator/cmd/automation"
```

Comment:
I know the tools repo already exists but as a future improvement it would be ideal if we could try to move these under the eng folder if possible.

Reply:
That is on our backlog to convert everything to powershell and move them to `eng` directory (track 1 related things excluded).
We are putting these here because we just want to implement this in the fastest way to make it running. We will translate this to powershell and move it to `eng` directory